support/reftest/data.yml

```diff
@@ -0,0 +1,22 @@
+prod-cdn-url:
+  https://cdn.redhat.com
+
+test_data:
+- path: /content/dist/rhel/server/7/7.4/x86_64/os/repodata/fd23895d43f54a50bbd0509809dd5f45298bfd6b-other.sqlite.bz2
+  sha256: 647b38c2222caecd0859f97a0ef0ef78f249b8d96af437552d6b2950056f261e
```

Comment:
Are these checksums guaranteed constant? There's no possibility that a the above path will be updated at any point?

Reply:
Most of them should never change.  'listing' files are the most likely exception here, I think, but there is no specific reason those here ought to be updated either (it's just that they "could" have formatting changes due to a new version of tools, or something like that).

The way I see it, the test data definition needs to balance between:

1. Don't define any checksums: then we don't really know what data we're testing, which means tests can't make any assumptions about it, so you can't directly encode an expected value of Digest headers for instance.
2. Define the checksums: then we know exactly our testdata, so our tests can be coded directly against that which allows them to be simple.  But, in exchange, if the source data changes, this config file needs maintenance.

I'm pretty comfortable with using (2) for everything we currently have in here, but I did think maybe we would have to end up with the script supporting both cases, with some data using (1) and some using (2).  e.g. a repomd.xml file seems inappropriate for (2) since any republish of the repo will change it.